src/neo/Helper.cs

```diff
@@ -310,5 +310,51 @@ internal static BigInteger WeightedAverage<T>(this IEnumerable<T> source, Func<T
                 yield return resultSelector(item, weight);
             }
         }
+
+        public static byte[] CommonPrefix(this byte[] a, byte[] b)
```

Comment:
This method can be removed.

Reply:
I need this method to get the common prefix between path and key.